dcat/index.html

```diff
@@ -46,37 +46,34 @@
-        <li>addition of a <a href="#Class:Resource"><code>dcat:Resource</code></a> class for representing any asset than can be included in the catalog, this is
-            now the super-class of <a href="#Class:Dataset"><code>dcat:Dataset</code></a></li>
-        <li>addition of <a href="#Class:Data_Service"><code>dcat:DataService</code></a>, as a sub-class of <a href="#Class:Resource"><code>dcat:Resource</code></a>, to support catalog service end-points providing access to data assets</li>
-        <li>addition of ways to represent <a href="#examples-bag-of-files">loosely structured catalogs</a>, where there is no distinction between a dataset and its distributions</li>
-        <li>more details for the ways of representing <a href="#examples-dataset-provenance">dataset provenance</a> and <a href="#quality-information">quality</a></li>
-        <li>an <a href="#dcat-sdo">alignment</a> between the DCAT vocabulary and the schema.org vocabulary</li>
+        <li>addition of <a href="#Property:distribution_checksum"><code>spdx:checksum</code></a> property and <a href="#Class:Checksum"><code>spdx:Checksum</code></a> class to provide digest for DCAT distributions</li>
+        <li>addition of properties for supporting versioning, e.g.,  <a href="#Property:resource_version"><code>dcat:version</code></a>, <a href="#Property:resource_previous_version"><code>dcat:version</code></a>, <a href="#Property:resource_has_current_version"><code>dcat:hasCurrentVersion</code></a>, see <a href="#dataset-versions"></a></li>
+        <li>addition of a <a href="#Class:Dataset_Series"><code>dcat:DatasetSeries</code></a> class and properties for representing Dataset Series, see <a href="#dataset-series"></a></li>
     </ul>
 
 
     <p>
         This new version of the vocabulary updates and expands the original but preserves backward compatibility. A full list of the significant changes (with links to the relevent github issues) is described in <a href="#changes"></a>.
     </p>
     <p>
-        The exit criteria for CR focussed on v2 new features that replicate features that were included in application profiles of v1 as a way of remedying missing and necessary elements.  The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-registries-version-meeting-september'>adopt the DCAT v2 model</a> in their work.
+        The exit criteria for CR focussed on v3 new features that replicate features that were included in application profiles of v2 as a way of remedying missing and necessary elements.  The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-registries-version-meeting-september'>adopt the DCAT v2 model</a> in their work.
```

Comment:
```suggestion
        The exit criteria for CR focussed on v3 new features that replicate features that were included in application profiles of v2 as a way of remedying missing and necessary elements.  <!--The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-registries-version-meeting-september'>adopt the DCAT v2 model</a> in their work.-->
        
As far as I know, there are not yet explicit commitments to update DCAT-AP wrt to DCAT3, although this is likely to happen.
```

Reply:
As far as I know, there are not yet explicit commitments to update DCAT-AP wrt to DCAT3, although this is likely to happen.